### PR TITLE
Remove itayd from CODEOWNERS

### PR DESCRIPTION
**Pull request by itayd**
Lets see if it solves some issues. Worst case I can override if needed.

### CODE REVIEW

**Comment by efiShtain** [comment]
I think a better approach for now is putting all of us as code owners
so any 2 people can approve a PR (one writes the code, the other one approves it)

or just remove the code owners for now and require at least one reviewer 

@itayd @daabr @ig-ra WDYT?

**Comment by ig-ra** [comment]
Agree. 2 approvals per PR

**Comment by daabr** [comment]
Let's discuss this in our next sync.

File: CODEOWNERS [hunk 1]
```diff
@@ -1,6 +1,5 @@
-* @itayd
 /cmd/ak/ @daabr
-/docs/ @daabr @haimzlato @itayd
-/backend/internal/db/ @efiShtain @itayd
-/integrations/ @daabr @itayd
+/docs/ @daabr @haimzlato
+/backend/internal/db/ @efiShtain
+/integrations/ @daabr
 /systest/ @daabr
```
Comment (daabr) [inline CODEOWNERS:5]:
```suggestion
/tests/system/ @daabr
```

File: CODEOWNERS [hunk 2]
```diff
@@ -1,6 +1,5 @@
-* @itayd
 /cmd/ak/ @daabr
-/docs/ @daabr @haimzlato @itayd
-/backend/internal/db/ @efiShtain @itayd
-/integrations/ @daabr @itayd
+/docs/ @daabr @haimzlato
```
Comment (daabr) [inline CODEOWNERS:2]:
```suggestion
```

**Pull request closed without merging**